<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RandSpgDialog</class>
 <widget class="QWidget" name="RandSpgDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>597</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Spacegroup Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <widget class="QPushButton" name="push_deselectAll">
     <property name="text">
      <string>Deselect all</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="push_decrementAll">
     <property name="text">
      <string>Decrement All</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="push_selectAll">
     <property name="text">
      <string>Select all</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="QTableWidget" name="table_list">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>95</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>23</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>23</number>
     </attribute>
     <column>
      <property name="text">
       <string>Space Group</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Possible Composition(s)</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Allow randSpg?</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Min xtals per Spg</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="push_incrementAll">
     <property name="text">
      <string>Increment All</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLabel" name="ui_label">
     <property name="text">
      <string>Empty</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>push_selectAll</zorder>
  <zorder>push_deselectAll</zorder>
  <zorder>table_list</zorder>
  <zorder>push_decrementAll</zorder>
  <zorder>push_incrementAll</zorder>
  <zorder>ui_label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
